extension/app/ts/background/background.ts

```diff
@@ -247,6 +245,8 @@ async function handleSigningMode(
 	try {
 		parsedRequest = EthereumJsonRpcRequest.parse(request.options)
 	} catch (error) {
+		console.log(request)
+		console.log(error)
```

Comment:
```suggestion
		console.error(request)
		console.error(error)
```

Reply:
What do you mean?  I don't think an app has insight into console errors.  The only thing this should do is change the color and how it gets filtered when using dev tooling.

Reply:
Chrome shows them as errors for the extension

Reply:
![image](https://user-images.githubusercontent.com/13102010/231687074-0f11c3ec-05f4-497a-819a-e14770316c6c.png)
this button appears